README.md

```diff
@@ -1,8 +1,12 @@
 [![Build Status](https://travis-ci.org/kodi-pvr/pvr.vuplus.svg?branch=master)](https://travis-ci.org/kodi-pvr/pvr.vuplus)
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/5120/badge.svg)](https://scan.coverity.com/projects/5120)
 
-# VuPlus PVR
-VuPlus PVR client addon for [Kodi] (http://kodi.tv)
+# Enigma2 PVR
+PVR client addon for [Kodi] (http://kodi.tv) to control Enigma2-based TV-receivers/DVR/set-top boxes. 
```

Comment:
https

Reply:
I have not changed the URL

Reply:
Yes, but the URL would be invalid. Can you change the http to https?

You could probably also do a single commit for these changes. Thanks